src/utils/themes.js

```diff
@@ -284,3 +288,20 @@ export function getThemeType(currentTheme: Theme = defaultTheme) {
 export function getThemeName(currentTheme: Theme = defaultTheme) {
   return currentTheme.current.replace('Theme', '');
 }
+
+export const getColorByTheme = (props: ColorsByThemeProps) => {
```

Comment:
Just making sure I understand: this function will be used instead of `themedColors` in styled components like this?
```js
const Root = styled.View`
  background-color: ${getColorByTheme({ lightKey: 'basic070', darkKey: 'basic070' })}
`
```
looks like a lot of typing :(

Reply:
Yeah, but just on cases where colors are not interchangeable.

Usually, this will be enought:
```
const Root = styled.View`
  background-color: ${({ theme }) => theme.colors.basic070};
`
```

And as we might have more themes in the future, I've added props as an object.

Reply:
that's better :D